src/main/resources/META-INF/resources/frontend/styles/styles.css

```diff
@@ -194,3 +194,11 @@
     display: flex;
     align-items: center
 }
+
+.margin-top {
```

Comment:
`Bootstrap` naming is good, but other classes do not use it. E.g. `.button-disabled` is not `.btn-disabled`. So it stay as it is, otherwise we should refactor other classes to go with Bootstrap naming